docker/package/model.py

```diff
@@ -633,28 +644,22 @@ def gen_spec_file(self, build_deps, run_deps, out):
         with open(out, "w") as f:
             f.write(file_contents)
 
-    def gen_makefile(self, out):
+    def gen_buildfile(self, out):
         file_contents = """
-.PHONY: install
-
-BINDIR=/usr/bin
-
-tezos-baking:
-
-tezos-setup-wizard:
-	mv $(CURDIR)/tezos_setup_wizard.py $(CURDIR)/tezos-setup-wizard
-	chmod +x $(CURDIR)/tezos-setup-wizard
-
-tezos-voting-wizard:
-	mv $(CURDIR)/tezos_voting_wizard.py $(CURDIR)/tezos-voting-wizard
-	chmod +x $(CURDIR)/tezos-voting-wizard
-
-install: tezos-baking tezos-setup-wizard tezos-voting-wizard
-	mkdir -p $(DESTDIR)$(BINDIR)
-	cp $(CURDIR)/wizard_structure.py $(DESTDIR)$(BINDIR)/wizard_structure.py
-	cp $(CURDIR)/tezos-setup-wizard $(DESTDIR)$(BINDIR)/tezos-setup-wizard
-	cp $(CURDIR)/tezos-voting-wizard $(DESTDIR)$(BINDIR)/tezos-voting-wizard
+from setuptools import setup
+
+setup(
+    name='tezos-baking',
```

Comment:
Currently `tezos-baking` python package has `0.0.0` version. However, it's possible to define version as an argument of `setup`, let's use the version from `self.meta` here to provide the actual version of the package